spec/factories/movie.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :movie do
+    name { "Title" }
```

Comment:
you can use [Faker gem](https://github.com/faker-ruby/faker) to generate random fake data.
[Here](https://github.com/faker-ruby/faker) you have available methods.

Reply:
Used faker methods for that a like this :D